connector_x/src/main.cpp

```diff
@@ -119,20 +116,20 @@ Adafruit_NeoPixel *getPixels(uint8_t port)
 {
     if (port == 0)
     {
-        return pixels0;
+        return pixels[0];
     }
 
-    return pixels1;
+    return pixels[1];
 }
 
 PatternRunner *getPatternRunner(uint8_t port)
 {
     if (port == 0)
```

Comment:
Why not check if the port is within the limit and then do `patternRunners[port]`?